recipes/optuna/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "optuna" %}
+{% set version = "0.18.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7f7d92d8d1a3e0c5c3b92b4272a29b110a4a3d53e140a55e564c8756225a0c99"
+
+build:
+  number: 0
+  entry_points:
+    - optuna = optuna.cli:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - alembic
```

Comment:
Are these all needed in host? Normally python and pip is enough

Reply:
You're right. Thank you for pointing it out. I removed unnecessary packages (https://github.com/conda-forge/staged-recipes/pull/10081/commits/f705b93abfee2adfbaccc2c1b8406385c50d075d).